src/neptune_fastai/impl/__init__.py

```diff
@@ -58,112 +58,81 @@
-            |--> loader
-            |--> optimizer hyperparameters
-        |--> ...
-        |--> fit_n
-
-    Note:
-        You can use public ``api_token="ANONYMOUS"`` and set ``project="common/fastai-integration"``
-        for testing without registration.
+    batch, and loader-level metrics.
+
+        metrics
+            |--> fit_0
+                |--> batch
+                |--> loader
+                |--> optimizer hyperparameters
+            |--> ...
+            |--> fit_n
+
+    Note: To try the integration without registering, you can use the public API token
+        `neptune.ANONYMOUS_API_TOKEN` and set the `project` argument to "common/fastai-integration".
 
     Args:
-        run (:obj: `neptune.new.run.Run`): Neptune run object
-            A run in Neptune is a representation of all metadata that you log to Neptune.
-            Learn more in `run docs`_.
-        base_namespace (:obj: `str`, optional): Root namespace. All metdata will be logged inside.
-        Defaults is empty string. In this case metadata is logged without common "base_namespace"
-        upload_saved_models (:obj: `str`, optional):  `'all'` or `'last'`.
-        When using `'all'`, it uploads all model checkpoints created by `SaveModelCallback()`.
-        When using `'last'`, it uploads the last model checkpoint created by `SaveModelCallback()`.
-        Defaults to `'all'`.
+        run: Neptune `Run` object. A run is a representation of all metadata that you log to Neptune.
+        base_namespace: Root namespace inside which all metdata will be logged.
+            If omitted, the metadata is logged without a common root namespace.
+        upload_saved_models: Which model checkpoints to upload.
+            - `"all"` (default): uploads all model checkpoints created by `SaveModelCallback()`.
+            - `"last"`: uploads the last model checkpoint created by `SaveModelCallback()`.
 
     Examples:
-        For more examples visit `example scripts`_.
 
-        Full script that does model training and logging of the metadata.
-            import fastai
-            from neptune.new.integrations.fastai import NeptuneCallback
-            from fastai.vision.all import *
+        Logging a single training phase:
+
+            from fastai.callback.all import SaveModelCallback
+            from fastai.vision.all import (
+                untar_data,
+                ImageDataLoaders,
+                ...
+            )
+
             import neptune.new as neptune
+            from neptune.new.integrations.fastai import NeptuneCallback
             from neptune.new.types import File
 
-            run = neptune.init(
-                project="common/fastai-integration", api_token="ANONYMOUS", tags="more options"
-            )
+            run = neptune.init_run()
 
             path = untar_data(URLs.MNIST_TINY)
-            dls = ImageDataLoaders.from_csv(path)
+            dls = ImageDataLoaders.from_csv(path, num_workers=0)
```

Comment:
I don't think we need to specify the `num_workers`, that's an individual choice. Plus if set to zero, it makes training significantly slower because loading data will take longer.